packages/shared/routes/dashboard/governance/views/GovernanceEventDetails.svelte

```diff
@@ -54,36 +57,49 @@
 <div
     on:click={handleBackClick}
     class="inline-flex justify-between items-center w-20 p-2 pr-4 bg-white hover:bg-gray-100 border
-    rounded-lg border-solid border-gray-300 cursor-pointer mb-5">
+    rounded-lg border-solid border-gray-300 cursor-pointer mb-5"
+>
     <Icon icon="arrow-left" classes="w-4 h-4 text-gray-500" />
     <Text type="p" smaller overrideColor classes="text-gray-800">{localize('actions.back')}</Text>
 </div>
 
 <div class="w-full h-full grid grid-cols-3 gap-x-4 min-h-0">
     <DashboardPane classes="w-full h-full p-6 col-span-2 flex flex-col">
-        <Text type="p" classes="mr-auto uppercase px-2 py-1 mb-2 text-blue-500 bg-blue-100 rounded-lg" smaller bold overrideColor>{event?.status?.status}</Text>
+        <Text
+            type="p"
+            classes="mr-auto uppercase px-2 py-1 mb-2 text-blue-500 bg-blue-100 rounded-lg"
+            smaller
+            bold
+            overrideColor>{event?.status?.status}</Text
+        >
         <Text type="h2" classes="mb-4">{event?.information?.name}</Text>
         <Text type="p" classes="mb-2">{event?.information?.additionalInfo}</Text>
         <Text type="p" classes="mb-2">{event?.information?.payload?.questions[0]?.text}</Text>
         <Text type="p" classes="mb-6">{event?.information?.payload?.questions[0]?.additionalInfo}</Text>
         {#each event?.information?.payload?.questions[0]?.answers as answer}
             <div
-                on:click={() => handleClick(answer)} 
-                class="py-4 px-6 bg-{isSelected(answer?.value) ? 'blue-100' : 'gray-50'} hover:bg-gray-100 border border-solid border-gray-100 rounded-lg flex justify-between mb-4 cursor-pointer"
+                on:click={() => handleClick(answer)}
```

Comment:
maybe we could convert this to a button and disable the possibility to click altogether if the event is not active

Reply:
instead of managing it on the `handleClick` function itself